src/App.vue

```diff
@@ -1,9 +1,13 @@
+<script setup lang="ts">
+import CitySearch from "@/components/city-search/CitySearch.vue";
+import BreadcrumbsBar from "@/components/breadcrumbs-bar/BreadcrumbsBar.vue";
+</script>
+
 <template>
-    <nav>
-        <router-link to="/">Home</router-link> |
-        <router-link to="/about">About</router-link> |
-        <router-link to="/dashboard">Dashboard</router-link>
-    </nav>
+    <div class="city-search-wrapper">
+        <CitySearch />
+        <BreadcrumbsBar />
```

Comment:
The breadcrumbsbar unfortunately disappears on mobile
<img width="439" alt="Bildschirmfoto 2024-01-30 um 19 03 46" src="https://github.com/sickinga/hux_weather/assets/48798335/8fdf4ae5-e446-465d-85cf-042a26c899cc">

Reply:
this is intentional because on mobile I thought it would be easier to use the standard mobile back